src/groups/mwc/mwcio/mwcio_ntcchannel.h

```diff
@@ -356,7 +356,8 @@ class NtcChannel : public mwcio::Channel,
     /// this channel for the write to succeed.
     void write(Status*            status,
                const bdlbb::Blob& blob,
-               bsls::Types::Int64 watermark) BSLS_KEYWORD_OVERRIDE;
+               bsls::Types::Int64 watermark = bsl::numeric_limits<int>::max())
+        BSLS_KEYWORD_OVERRIDE;
```

Comment:
https://github.com/bloomberg/blazingmq/blob/50018e73fece05e5bebcb2f3ad3b1e31d49e1c17/src/groups/mwc/mwcio/mwcio_channel.h#L178-L181
The interface has default for warermark